atlas/pool.go

```
@@ -15,29 +16,54 @@
+	amu := sync.Mutex{}
+
 	Pool = sqlitemigration.NewPool(options.DbFilename, sqlitemigration.Schema{}, sqlitemigration.Options{
 		Flags:    sqlite.OpenReadWrite | sqlite.OpenCreate | sqlite.OpenWAL,
 		PoolSize: runtime.NumCPU() * 2,
 		PrepareConn: func(conn *sqlite.Conn) (err error) {
-			// todo: err = conn.SetAuthorizer(authPrinter{})
-			err = conn.SetDefensive(true)
+			auth := &Authorizer{}
+
+			amu.Lock()
+			authorizers[conn] = *auth
```

golangci-lint (1.62.2)

copylocks: assignment copies lock value to authorizers[conn]: github.com/bottledcode/atlas-db/atlas.Authorizer contains sync.RWMutex

(govet)